test/RoaveTest/SecurityAdvisories/VersionConstraintTest.php

```diff
@@ -255,8 +252,16 @@ public function closedRangesProvider(): array
             ['>1-stable.1.2,<1-rc.1.2'],
         ];
 
-        return array_combine(
-            array_column($samples, 0),
+        return Dict\associate(
+            Vec\map(
+                $samples,
+                /**
```

Comment:
I think we should look at examples where `/** @var` was **added** mid-code, and revert those